tests/components/template/test_light.py

```diff
@@ -607,6 +672,7 @@ async def test_level_action_no_template(
             "{{ state_attr('light.nolight', 'brightness') }}",
             ColorMode.BRIGHTNESS,
         ),
+        (None, "{{'one'}}", ColorMode.BRIGHTNESS),
```

Comment:
Why `{{'one'}}`?

Reply:
This was part of the 'test coverage' improvement.
`self._update_brightness` function test for `None`, `"None"`, `""` and int value.
i added this to test the except at +681
```python
        except ValueError:
            _LOGGER.exception(
                "Template must supply an integer brightness from 0-255, or 'None'"
            )
            self._brightness = None
```
If a template return a string who can't be converted to an int, like a spelled out value, it run this code and return `None`
The same have been done for many other test:
- test_temperature_template
- test_legacy_color_template
- test_hs_template
- test_rgb_template
- test_rgbw_template
- test_rgbww_template